### PR TITLE
 Add Luna Project Office card

### PR DESCRIPTION
**Pull request by kberg**
Has a nice pair of tests that go through the draft phase, so that's helpful.

### CODE REVIEW

**Comment by kberg** [comment]
Interesting! A flaky test. I'm on it.

**Comment by kberg** [comment]
Figured out what was making this test fail.

File: src/cards/moon/LunaProjectOffice.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import {Player} from '../../Player';
+import {CardType} from '../CardType';
+import {IProjectCard} from '../IProjectCard';
+import {Tags} from '../Tags';
+import {CardMetadata} from '../CardMetadata';
+import {CardRenderer} from '../render/CardRenderer';
+import {Cards} from '../Cards';
+import {MoonExpansion} from '../../moon/MoonExpansion';
+
+export class LunaProjectOffice implements IProjectCard {
```
Comment (sienmich) [inline src/cards/moon/LunaProjectOffice.ts:11]: I think the new cards should be made static. `export class LunaProjectOffice extends Card implements IProjectCard`

Comment (kberg) [reply src/cards/moon/LunaProjectOffice.ts:11]: Oh that was an oversight on my part. Thanks.

**Review by sienmich — APPROVED** [review]

**Pull request merged**